LogicLayer/GameEventParticipations/GameEventParticipationService.cs

```diff
@@ -85,8 +85,6 @@ public async Task ExitGameEventAsync(int gameEventId, string userName)
                 case ParticipationStatus.Creator:
                     throw new GameEventParticipationException("As a creator, you cannot exit your own event.");
                 case ParticipationStatus.PendingGuest:
-                case ParticipationStatus.ExitedGuest:
```

Comment:
This diff looks strange, but ExitedGuest is not being processed inside this switch statement.

Reply:
Sorry, I didn't notice that there are two of these. The other switch statement above doesn't process ExitedGuest either. Actually this one doesn't process ExitedGuest and PendingGuest and it seems you did it intentionally. I need an explanation, because I thought we were supposed to correct things underlined by R#.

Reply:
Neither of these should be returned by `GetActiveGameEventParticipation`, so they hit the `default` and throw an exception. R# isn't all-knowing and this is only a hint. I added a comment since this is not obvious.